cypress/page-objects/app/datasets-page.ts

```diff
@@ -36,3 +36,7 @@ class DatasetPage extends BasePage {
 }
 
 export const datasetPage = new DatasetPage()
+
+function addDataId(selector: string): string {
```

Comment:
No need to add this function to every page. You can just create a separate class `Helpers` and add this method there and reuse in all pages via import. I'd rename it to `getDataId`.